lib/zendesk_apps_support/version.rb

```diff
@@ -1,3 +1,3 @@
 module ZendeskAppsSupport
-  VERSION = "4.43.1"
+  VERSION = "4.44.0"
```

Comment:
Hi @zendesk-punavwalke ,

This version should be only released when this gem is stable with all the necessary updates. I'd recommend releasing `4.44.0.alpha.1` to integrate this first pre-release version with ZAM and closing this PR. Please follow the instructions in [README.md](https://github.com/zendesk/zendesk_apps_support/blob/main/README.md)